lib/vsc/install/shared_setup.py

```diff
@@ -1556,7 +1581,8 @@ def build_setup_cfg_for_bdist_rpm(target):
             txt.extend(["build_requires = %s" % (klass.sanitize(target['setup_requires']))])
 
         # add metadata
-        txt += ['', '[metadata]', '', 'description-file = %s' % README, '']
+        description_file = target.get('description_file', 'Unknown')
```

Comment:
`Unknown`? why is this needed?

Reply:
Just a precaution in case `build_setup_cfg_for_bdist_rpm` would be called without going through `action_target` and `description-file` would be unset.

Reply:
if there is none, don't add the metadat lines in setup.cfg. but i prefer hard failure, people who can't be bothered to add simple Readme, well, <insert swearing of your choice> them.

Reply:
ok, added https://github.com/hpcugent/vsc-install/pull/176/commits/dbea7c7f9447d8a71b0bf77b2eb2b89b88a348a5 to only set the metadata if `description_file` is set. On the other hand, the hard failure if README is missing is already there. It happens before this point, in `locate_readme` when its called from `action_target`, line 1350. So if there is no README, you don't get here.